mantis-server/mantis-server-agent/src/main/java/io/mantisrx/server/agent/TaskExecutorStarter.java

```diff
@@ -129,6 +132,11 @@ public TaskExecutorStarterBuilder rpcSystem(RpcSystem rpcSystem) {
             return this;
         }
 
+        public TaskExecutorStarterBuilder registry(MeterRegistry meterRegistry) {
```

Comment:
Is this method called anywhere? I don't think we'll pass `MeterRegistry` in `TaskExecutorStarter`.
Note to self: Investigate this more!